front/src/modules/trainschedule/components/Timetable/TimetableTrainCard.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
-import { FaTrash, FaPencilAlt } from 'react-icons/fa';
+import { GoPencil, GoTrash } from 'react-icons/go';
```

Comment:
```suggestion
import { GoCopy, GoPencil, GoTrash } from 'react-icons/go';
```
GoCopy can be used instead of `MdContentCopy` at **line 292**